future-apps/payroll/package.json

```diff
@@ -59,9 +59,9 @@
     "@aragon/cli": "~5.6.0",
     "@aragon/test-helpers": "^2.0.0",
     "eth-gas-reporter": "^0.2.0",
-    "ethereumjs-testrpc-sc": "^6.1.6",
+    "ethereumjs-testrpc-sc": "^6.4.5-sc.3",
```

Comment:
since the new `solidity-coverage` version is using ganache, do we still need this one?

Reply:
I think so, the reason was that `lerna` hoisting makes running `testrpc-sc` a bit of a pain unless we explicitly declare this dependency here.

Reply:
I see, but I we shouldn't need to run it anymore right?

Reply:
I think we still have to; the `ganache-cli.sh` script runs `testrpc-sc` and AFAIK coverage still requires the modified version of testrpc.